### PR TITLE
Update CVE-2023-30536

### PR DESCRIPTION
**Pull request by williamdes**
Ref: https://github.com/slimphp/Slim-Psr7/issues/284#issuecomment-1541328898

### CODE REVIEW

**Comment by Ocramius** [comment]
This repository only contains the list of generated conflict ranges (happens automatically).

If the security issue has been fixed in other versions too, then https://github.com/advisories/GHSA-q2qj-628g-vhfw needs updating: to do that, you need to probably contact github support.

**Comment by williamdes** [comment]
> This repository only contains the list of generated conflict ranges (happens automatically).
> 
> If the security issue has been fixed in other versions too, then [GHSA-q2qj-628g-vhfw](https://github.com/advisories/GHSA-q2qj-628g-vhfw) needs updating: to do that, you need to probably contact github support.

Thanks, already done: https://github.com/github/advisory-database/pull/2233#pullrequestreview-1420088646

So let's wait ;p
Maybe document this on the README ?

**Comment by Ocramius** [comment]
> Maybe document this on the README ?

Do send a patch :)

File: composer.json [hunk 1]
```diff
@@ -411,7 +411,7 @@
         "simplesamlphp/simplesamlphp-module-openidprovider": "<0.9",
         "simplito/elliptic-php": "<1.0.6",
         "sitegeist/fluid-components": "<3.5",
-        "slim/psr7": "<1.6.1",
+        "slim/psr7": "<1.4.1|<1.5.1|<1.6.1",
```
Comment (Ocramius) [inline composer.json:414]: BTW, this range is insufficient: this still is equivalent to just `<1.6.1`.

**Comment by williamdes** [comment]
> > Maybe document this on the README ?
> 
> Do send a patch :)

Too much things I never do, will not have the time. Sorry for that

File: composer.json (hunk 1 above)
Comment (williamdes) [reply composer.json:414]: Yeah, it the right one is https://github.com/github/advisory-database/pull/2233#discussion_r1189601026

**Pull request closed without merging**